Kitsune/ClubPenguin/Login.php

```diff
@@ -27,14 +27,15 @@ protected function handleLogin($socket) {
 		$password = Packet::$Data['body']['login']['pword'];
 		
 		if($penguin->database->usernameExists($username) === false) {
+			password_verify('KitsuneTimingAttack', '$2y$12$5hf7vOtBjsNJf6oZSsnyXOlcLTRblTwSg550SR0ohxVUkItmXhMM6'); //Mitigate timing attacks
```

Comment:
Props for thinking to add a safeguard against timing attacks, but [password_verify() already implements a constant time xor expression](https://github.com/php/php-src/blob/9b46a15a6e82f7f1fb3b921b44b89cd74dd0839e/ext/standard/password.c#L273) which serves the same purpose as this call.